src/main/java/com/intuit/graphql/orchestrator/resolverdirective/FieldResolverDirectiveUtil.java

```diff
@@ -124,37 +125,35 @@ public static boolean isObjectOrInterfaceType(TypeDefinition typeDefinition) {
 
   public static List<FieldResolverContext> createFieldResolverContexts(
       TypeDefinition typeDefinition, XtextGraph xtextGraph) {
-
     List<FieldDefinition> childFields = getFieldDefinitions(typeDefinition);
-    return childFields.stream()
-        .map(childFieldDefinition -> {
-          List<Directive> directives = getResolverDirective(childFieldDefinition);
-
-          if (CollectionUtils.size(directives) > 1) {
```

Comment:
this check will not go away

Reply:
can we retain the existing spacing, also use Google Style.